### PR TITLE
Add fresh new styles for TOC pages

### PR DESCRIPTION
**Pull request by culas**
I built the TOC based on the previous one. It would look like this:

image

There is a new class "counter" that can be put on the chapter titles to have automatic numbering. The first two chapters in the screenshot are with this class. It also adds a nice purple line.
It's a class to make it backwards compatible.

### CODE REVIEW

File: src/styles/zuehlke.css [hunk 1]
```diff
@@ -17,10 +17,16 @@
     --zuehlke-blue: #0099cc;
     --zuehlke-bright-green: #ccff00;
     --zuehlke-green: #00cc66;
+    --zuehlke-gradient: linear-gradient(to right bottom, var(--zuehlke-purple), var(--zuehlke-bright-blue));
```
Comment (romanutti) [inline src/styles/zuehlke.css]: Quickly checked the gradient in the the slidedecks master view - would be the following:

```suggestion
    --zuehlke-gradient: linear-gradient(to right bottom, var(--zuehlke-purple) 10%, #A1C2E6);
```

File: src/styles/zuehlke.css [hunk 2]
```diff
@@ -387,7 +393,7 @@ body {
 .reveal .hl {
     color: transparent;
     font-weight: bold;
-    background: linear-gradient(to right bottom, var(--zuehlke-purple), var(--zuehlke-bright-blue));
+    background: var(--zuehlke-gradient);
```
Comment (romanutti) [inline src/styles/zuehlke.css:403]: 👏🏼

File: src/styles/zuehlke-toc.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    color: inherit;
+}
+
+.toc section h2 {
+    margin: 0;
+    padding: 2px 12px 8px;
+}
+
+.toc section h2.counter:before {
+    content: counter(chapterCounter, decimal-leading-zero);
```
Comment (romanutti) [inline src/styles/zuehlke-toc.css:71]: 🤯 

Nice!

**Comment by romanutti** [comment]
> I built the TOC based on the previous one. It would look like this:
> 
> image
> 
> There is a new class "counter" that can be put on the chapter titles to have automatic numbering. The first two chapters in the screenshot are with this class. It also adds a nice purple line. It's a class to make it backwards compatible.

Really cool stuff Lucas, thanks a lot! 👏🏼 

Generally, I tried to keep the designs as close as possible to the slides from templafy. This will make it easier to keep a consistent design - also having in mind that other may contribute in the future as well.

Some (more and some less similar) suggestions, both based on "existing slides" from templafy:

TOC-a

TOC-c

image

What is your take on that? 😃

File: src/styles/zuehlke.css (hunk 1 above)
Comment (romanutti) [reply src/styles/zuehlke.css]: Updated it in pr https://github.com/webplatformz/zuehlke-reveal-package/pull/4 as well

**Comment by culas** [comment]
> Really cool stuff Lucas, thanks a lot! 👏🏼
> 
> Generally, I tried to keep the designs as close as possible to the slides from templafy. This will make it easier to keep a consistent design - also having in mind that other may contribute in the future as well.
> 
> Some (more and some less similar) suggestions, both based on "existing slides" from templafy: [omitted from quote]
> 
> What is your take on that? 😃

I tried to make it as close to the previous design, but I really like the second slide you suggested. It has the same structure/layout and has a large coloured area to make it pop. I'll adjust the design.

**Comment by culas** [comment]
New layout done ("Strings" has hover effect): 
image

**Review by romanutti — APPROVED** [review]

**Pull request merged**